src/index.vue

```diff
@@ -183,6 +183,14 @@ export default {
     getTranslation (key) {
       const locale = this.getLocale()
       return get(messages, `${locale}.${key}`)
+    },
+    documentClick (e) {
```

Comment:
As discussed in person, it would be great to have a more descriptive name such as `closeOnOutsideClick` or something like that.